gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/schema/FeatureConverter.java

```diff
@@ -75,18 +76,19 @@ private static PolygonFeature toFeatureFromGeometry(
       throw new DataStoreException("Unknown geometry type in feature " + doc.getId() + ": " + type);
     }
     Object coordinates = geometry.get(GEOMETRY_COORDINATES);
-    if (coordinates == null || !coordinates.getClass().isArray()) {
+    if (coordinates == null || !(coordinates instanceof ArrayList)) {
```

Comment:
Do we care what specific implementation of `List` the coordinates are returns as? If not, then just check `instanceof List`.

Reply:
replaced with list.